crates/router/src/routes/app.rs

```diff
@@ -764,9 +764,12 @@ impl PaymentMethods {
                 .service(
                     web::resource("/{payment_method_id}")
                         .route(web::get().to(payment_method_retrieve_api))
-                        .route(web::post().to(payment_method_update_api))
                         .route(web::delete().to(payment_method_delete_api)),
                 )
+                .service(
+                    web::resource("/{payment_method_id}/update")
```

Comment:
We're sure this won't affect any sbx/prod merchants right?

Reply:
Yes, currently this api endpoint is unused